documentation/slate/source/includes/welcome.md

```diff
@@ -98,7 +98,9 @@ To help you in the building of your project, we have implemented tools to query
 About the data
 --------------
 
-The street network is extracted from [OpenStreetMap](http://www.openstreetmap.org). The public transport data are provided by networks that provide their timetables as open data. Some data improvements are achieved by Kisio Digital and are published back there <https://navitia.opendatasoft.com>
+The street network is extracted from [OpenStreetMap](http://www.openstreetmap.org). The public transport data are provided by networks that provide their timetables as open data. Some data improvements are achieved by Kisio Digital and are published back there <https://navitia.opendatasoft.com>.
+
+Want to know if your city is in Navitia? Know if a special contributor is used? You can either search in [datasets](#datasets) of the different [coverages](#coverage). Or use the filter provided on our data catalog <https://navitia.opendatasoft.com>.
 
```

Comment:
Tx a lot for that :)
Maybe add a Playground link in the third column?
http://canaltp.github.io/navitia-playground/play.html?request=https%3A%2F%2Fapi.navitia.io%2Fv1%2Fcoverage%2Fsandbox%2Fcontributors%2F%2F%3F&token=3b036afe-0110-4202-b9ed-99718476c2e0